lib/index.js

```diff
@@ -179,4 +193,21 @@ module.exports = class EyeglassCompiler extends BroccoliSassCompiler {
 
     return key + "+" + dependencies;
   }
+
+  cacheAssetImports(collection) {
+    let self = this;
+    let realAsAssetImport = collection.asAssetImport;
+    collection.asAssetImport = function(name) {
+      // if this has already been generated, return it from cache
+      var cacheKey = collection.cacheKey(name);
+      if (self.assetImportCache[cacheKey] !== undefined) {
```

Comment:
sometimes it is more appropriate for this cache check to be `cacheKey in self.assetImportCache`, this then also supports the cache containing `undefined` values. If that isn't applicable, feel free to disregard.

Reply:
this won't cache anything using an `undefined` key